pages/about.tsx

```diff
@@ -122,15 +124,77 @@ export default function AboutPage() {
                 <MuiMarkdown>{content.joinUs.body}</MuiMarkdown>
               </Typography>
               <Stack sx={{ direction: 'column' }}>
-                <Button href="/donate" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[3].name}</Button>
-                <Button href="/get-involved" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[4].name}</Button>
+                <Button href="/donate" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[5].name}</Button>
+                <Button href="/get-involved" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[6].name}</Button>
               </Stack>
             </Grid>
           </Grid>
         </Box>
       </SectionContainer>
 
-      {aboutContent.ourTeam.map((item) => (
+      <SectionContainer id="our-partners" sx={{ margin: 'auto', mb: 4, maxWidth: '65rem' }} maxWidth={false}>
+        <Typography variant="h2" sx={{ my: '3rem' }}>{content.ourPartners.header}</Typography>
+        <PaperSection title="" sx={{ pt: 0, pb: 1 }}>
+          <Grid container>
+            <Grid item xs={12}>
+              <Box display="flex" justifyContent="center" sx={{ pt: 4 }}>
+                <ImageContainer
+                  alt=""
+                  src={content.ourPartners.imgsrc as string}
+                  width={700}
+                  height={450}
+                  useImageDimensions
```

Comment:
I am not sure that you want to be using this, because if true (as it is), this will ignore dimensions passed in and use the true image dimensions.